contracts/axelar-gateway/src/auth.rs

```diff
@@ -104,7 +105,11 @@ pub fn rotate_signers(
         &new_epoch,
     );
 
-    event::rotate_signers(env, new_epoch, new_signers_hash);
+    event::SignersRotatedEvent {
+        epoch: new_epoch.clone(),
+        signers_hash: new_signers_hash.clone(),
```

Comment:
Please avoid using `.clone()` if possible, it's redundant here.

```suggestion
        epoch: new_epoch,
        signers_hash: new_signers_hash,
```